.gitmodules

```diff
@@ -1,4 +1,5 @@
 [submodule "monero"]
 	path = monero
-	url = https://github.com/monero-project/monero
+	url = https://github.com/stoffu/monero
```

Comment:
This will be replaced by the official repository URL once the rebase patches are merged.